jest.config.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  preset: 'jest-preset-angular',
+  setupFilesAfterEnv: ['<rootDir>/setup-jest.ts'],
+  globals: {
+    "ts-jest": {
+      "tsconfig": "<rootDir>/projects/demo-app/tsconfig.spec.json"
```

Comment:
Why the root config depends on projects/demo-app? If I delete the `demo-app` project will it break the tests configuration for other projects?

Reply:
I didn't find the `tsconfig.spec.json` file in the root folder. To fix it - we can add `tsconfig.spec.json` file to the root and change `tsconfig` as  `"tsconfig": "<rootDir>/tsconfig.spec.json"`